### ISSUE TITLE
P2P > gotortc streaming crashed Eufy Security integration (v8.0.1). Streaming non-functional after restart

### ISSUE THREAD

**Issue by V4ler1an** [fuatakgun/eufy_security#1085]
Following integration update to v8.0.1 I activated a P2P camera stream and left it running on screen. The stream stopped after 22 minutes and the Eufy Security integration crashed reporting these errors:
![image](https://github.com/fuatakgun/eufy_security/assets/134781183/c53fc473-78b6-4f8f-aefe-06b82116ccca)
![image](https://github.com/fuatakgun/eufy_security/assets/134781183/bbddfa91-57af-4eeb-b776-5bd580af612b)

The watchdog failed to restart the integration so it has to be done manually. However, following the restart the stream would no longer display in gotortc or the webrtc card. Even after a full server reboot. Issue affects all cameras.
![image](https://github.com/fuatakgun/eufy_security/assets/134781183/55b31d9b-ad36-4671-8adc-e1e41787eaff)

**To reproduce**

1. Update to v8.0.1
2. Go to 'Start P2P Stream'
3. Click on 'Display the stream in GoToRTC or WebRTC card'
4. Wait for the stream to cut and integration crashes.
5. Restart everything then try to start the stream again.

**Expected behavior**

Continued streaming.

**Additional information**

Go to Settings -> System -> Repairs -> Click on 3 dots (...) -> System Information and get Version and Installation Type to below fields;
- Home Assistant Installation Type (OS - Supvervised - Core): Home Assistant OS
- Home Assistant Core Version:  core-2024.1.2
- Eufy Security Add-on Version: 1.7.1
- Eufy Security Integration Version: 8.0.1

Hardware Information;
- Camera Model: S100
- Live Streaming Protocol (RTSP/P2P):  P2P > RTSP (GoToRTC displayed in either browser or WebRTC lovelace card)
- Debug Logs from Home Assistant (https://github.com/fuatakgun/eufy_security/discussions/624) - Without this, I will automatically resolve this issue:
Debug logs attached.
- Logs from Eufy Security Add-on (either from Add-ons page of Home Assistant or Docker output): 
Add-on crashes and
[Eufy security debug.txt](https://github.com/fuatakgun/eufy_security/files/13932692/Eufy.security.debug.txt)
 doesn't log any errors.
- Would you share your device with me in case further debugging required? (Yes/No): Yes

**Additional context**

Debug logs attached.

**Comment by hillshome** [comment]
I get the exact same. Frustrating!!!

**Comment by fuatakgun** [comment]
Guys, come on, what is the frustration? Did Eufy promise you that someone will support home assistant integration and now you are getting frustrated???

**Comment by V4ler1an** [comment]
Yes, quite. Please keep some perspective. Your efforts are hugely appreciated, thank you. Eufy is quite a sizeable and popular ecosystem and this integration is very valuable to the Home Assistant community.

**Comment by rodgers86** [comment]
I had a similar issue yesterday I believe. But it seems to have resolved itself?

**Comment by alexeiw123** [comment]
I'm seeing streaming and floodlight errors after a HA reboot. Restarting the add-on seems to resolve it. I only noticed this after 8.0.1. Not frustrated at all - I'm amazed at the work that's been done on this ecosystem.

**Comment by philippk75** [comment]
idem for me, the addon crashed a few time a day :-(

**Comment by duffy6** [comment]
same over here
